### ISSUE TITLE
Can't login to Microsoft account [BUG] <Short description>

### ISSUE THREAD

**Issue by Serverport** [PojavLauncherTeam/PojavLauncher#2511]
### Describe the bug

If you try to login it keeps saying error error error game crash     even i can't sign in with my official Java account 

I think the developer force coded the launcher to act like this 

### The log file and images/videos

![Screenshot_2022_0106_095726](https://user-images.githubusercontent.com/97210183/148328116-454b5126-c4c9-44b9-9224-ec2068608087.jpg)

### Steps To Reproduce

```markdown
Just try logging in
```

### Expected Behavior

Game crash or error

### Platform

```markdown
- Device model: vivo y11
- CPU architecture: aarch64
- Android version: 9
- PojavLauncher version: latest
```

### Anything else?

Nothing

**Comment by khanhduytran0** [comment]
> I think the developer force coded the launcher to act like this

I think you didn’t read the source code. Also you typed Microsoft login credentials into Mojang login boxes.

**Comment by khanhduytran0** [comment]
> Also you typed Microsoft login credentials into Mojang login boxes.

The big Microsoft login button is literally at the bottom but you ignored it to type into boxes that are supposed for Mojang account.

**Comment by Serverport** [comment]
> 

I know but it's happening again and again even can't login with Mojang or any other account if I try login with Microsoft this error will come java.lang.RuntimeException: 

MSA Error: 401: Unauthorized, error stream:
{"Identity":"0","XErr":2148916233,"Message":"","Redirect":"https://start.ui.xboxlive.com/CreateAccount"}
	at net.kdt.pojavlaunch.authenticator.microsoft.Msa.throwResponseError(Msa.java:293)
	at net.kdt.pojavlaunch.authenticator.microsoft.Msa.acquireXsts(Msa.java:169)
	at net.kdt.pojavlaunch.authenticator.microsoft.Msa.acquireXBLToken(Msa.java:122)
	at net.kdt.pojavlaunch.authenticator.microsoft.Msa.acquireAccessToken(Msa.java:75)
	at net.kdt.pojavlaunch.authenticator.microsoft.Msa.<init>(Msa.java:35)
	at net.kdt.pojavlaunch.authenticator.microsoft.MicrosoftAuthTask.doInBackground(MicrosoftAuthTask.java:72)
	at net.kdt.pojavlaunch.authenticator.microsoft.MicrosoftAuthTask.doInBackground(MicrosoftAuthTask.java:22)
	at android.os.AsyncTask$2.call(AsyncTask.java:333)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:245)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:764)


Blah blah blah blah blah

**Comment by khanhduytran0** [comment]
> I know but it's happening again and again even can't login with Mojang or any other account

Where did you get them? :eyes:

> https://start.ui.xboxlive.com/CreateAccount

Did you create an account + buy the game? seems not

**Comment by Serverport** [comment]
> > I know but it's happening again and again even can't login with Mojang or any other account
> 
> Where did you get them? 👀
> 
> > https://start.ui.xboxlive.com/CreateAccount
> 
> Did you create an account + buy the game? seems not

I get the account from my friend he has one main Java account and alt account 🙄

**Comment by ghost** [comment]
> > > I know but it's happening again and again even can't login with Mojang or any other account
> > 
> > 
> > Where did you get them? 👀
> > > https://start.ui.xboxlive.com/CreateAccount
> > 
> > 
> > Did you create an account + buy the game? seems not
> 
> I get the account from my friend he has one main Java account and alt account 🙄

@khanhduytran0 duyduyduyduyduduyduy NFA USER DETECTED

**Comment by khanhduytran0** [comment]
> I get the account from my friend he has one main Java account and alt account 🙄

> with <ins>my</ins> official Java account

Then why did you say that it was your account?
Anyways, we don’t support pirating like that.

**Comment by Serverport** [comment]
> > I get the account from my friend he has one main Java account and alt account 🙄
> 
> > with my official Java account
> 
> Then why did you say that it was your account? Anyways, we don’t support pirating like that.
Btw what I am saying *FIX* THIS BUG otherwise sell this launcher to someone who takes care of it 😐

**Comment by artdeell** [comment]
Do you think we are stupid enough to sell it to somebody? You can't get profit from it so you can't really find buyers anyway.

**Comment by NotAHero04** [comment]
block moment? :/

**Comment by Serverport** [comment]
Do you think we are stupid enough to sell it to somebody? 


MAYBE 🙄

**Comment by Serverport** [comment]
When I get a crash if I check the crash log there is a sentence that make a TRUE Minecraft player ANGRY 





*HI I AM MINECRAFT I AM A CRASHOLIC*



THIS IS how you devlop this launcher

**Comment by Serverport** [comment]
> block moment? :/

Yes

**Comment by ENDERMANYK** [comment]
Imagine "fix" a bug related to ~~NFA account and Minecraft itself~~

**Comment by khanhduytran0** [comment]
> Btw what I am saying FIX THIS BUG otherwise sell this launcher to someone who takes care of it

We aren’t Microsoft, you know. Minecraft is still owned by Microsoft, not us.

**Comment by khanhduytran0** [comment]
Locking as this issue is going to be overheat.

**Comment by khanhduytran0** [comment]
> FIX THIS BUG 

You’d better to buy the game, the bug fix costs $26.95

**Comment by khanhduytran0** [comment]
> When I get a crash if I check the crash log there is a sentence that make a TRUE Minecraft player ANGRY 
>
> *HI I AM MINECRAFT I AM A CRASHOLIC*
>
> THIS IS how you devlop this launcher 

Imagine expecting a PC game to fully functional on mobile. It's still a PC game, **we don't develop the game itself**. As I said, the game is still owned by Mojang/Microsoft. It has never been designed for mobile.
